### PR TITLE
Update Alarm.com Alarm Control Panel configuration variable

### PR DESCRIPTION
**Pull request by klaasnicolaas**
Update style of Alarm.com Alarm Control Panel documentation to follow new configuration variables description.
Related to #6385.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/alarm_control_panel.alarmdotcom.markdown [hunk 1]
```diff
@@ -24,9 +24,22 @@ alarm_control_panel:
...
+  type: string
+password:
+  description: Password for Alarm.com account.
+  required: true
+  type: string
+name:
+  description: The name of the alarm.
+  required: false
+  default: Alarm.com
+  type:
```
Comment (WofWca) [inline source/_components/alarm_control_panel.alarmdotcom.markdown]: Here

Comment (klaasnicolaas) [reply source/_components/alarm_control_panel.alarmdotcom.markdown]: Whoop i see, forgot the type 😋

**Comment by klaasnicolaas** [comment]
Sorry for the two concord232 commits, was not the intention. (Newbee in town 😋 )

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @klaasnicolaas!

Don't worry about the extra commits, we'll squash them away 😉 
Everybody has to learn, right? 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Comment by klaasnicolaas** [comment]
Thnx @frenck!

If there are still things that I have to pay extra attention to, you have to say it 😉

**Comment by frenck** [comment]
@klaasnicolaas You are doing fine! 👍

**Pull request merged**